www/js/charts/histogramChartDirective.js

```diff
@@ -140,8 +140,16 @@ export default ($log) => {
             // Frequency for how tall each bar is on left Y-Axis
             let frequency = data.map((d) => { return d.frequency; }),
                 maxFrequency = max(frequency),
-                minFrequency = min(frequency),
-                barHeight = barContainerHeight / maxFrequency;
+                minFrequency = min(frequency);
+
+            let domainMax = max([max(frequency), ticks]),
+                remainder = domainMax % ticks;
+
+            if (remainder > 0 || domainMax === max(frequency)) {
```

Comment:
additional conditional handles max(freq) values less than ticks, and domainMax == max(freq)